src/Events/Tap/TappedEntityGroup.ts

```diff
@@ -77,6 +81,9 @@ export const tappedEntityGroup = ({
       action = ActionType.tappedArtistGroup
       break
     case OwnerType.artwork:
+    case OwnerType.category:
```

Comment:
In other parts of the schema we use `gene` rather than category - I defer to the data team on their preference but should be consistent.

Reply:
in iOS at least, we transform the `Category` screen into a `gene` pagetype so ultimately it won't matter in redshift, but I think if `gene` is used throughout other parts of the schema, that's what we should use here too. 

To be clear, these describe the following page, right?
![IMG_5666](https://user-images.githubusercontent.com/26879583/81842855-bebd8080-951a-11ea-989e-17781bdd8046.PNG)

Reply:
Correct that is the page 👍